src/toolkit/facility_cost.cc

```diff
@@ -0,0 +1,28 @@
+#include "facility_cost.h"
+
+#include <math.h>
+#include <stdio.h>
+#include <iomanip>
+#include <sstream>
+
+namespace cyclus {
+namespace toolkit {
+
+// Interested in thoughts on the member initializer list
+FacilityCost::FacilityCost() : capital_cost_(0) {}
+
+FacilityCost::FacilityCost(double capital_cost) {
+  capital_cost_ = capital_cost;
+}
+
+FacilityCost::~FacilityCost() {}
```

Comment:
This can be defined in the header.  It doesn't actually need to be defined since it has default behavior